### PR TITLE
Feature/1.5

### PR DESCRIPTION
**Pull request by anshul-tii**
A script to enable connectivity with a desired AP

### CODE REVIEW

File: common/scripts/mesh-11s.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ network={
     psk="$5"
     key_mgmt=SAE
     ieee80211w=2
-    mesh_fwding=0
```
Comment (joenpera) [inline common/scripts/mesh-11s.sh:110]: "mesh_fwding=0" configuration can be used with our wpa_supplicant.  If this is commented out, it breaks other projects which are using common scripts.

**Review by joenpera — CHANGES_REQUESTED** [review]
Just checking the common scripts part.

File: common/scripts/mesh-11s.sh (hunk 1 above)
Comment (joenpera) [reply common/scripts/mesh-11s.sh]: af3cfe5 fixes this one.

**Comment by anshul-tii** [comment]
Still not integrated on develop

**Pull request closed without merging**